README.md

```diff
@@ -132,6 +132,8 @@ Runs ez-build in production mode, which implies a higher optimization level (cur
 Toggles flags that may affect the output or behavior of ez-build. The available options are:
 
   - `add-module-exports:<true|false>` toggles whether the UMD output of ez-build should be backwards compatible with AMD and CJS module formats. If this setting is `true`, ez-build will ensure any module with a single `export default` will not export an object with a `default` key. This value defaults to `false`. It is only recommended you use this flag if you *must* keep backwards compatibility with legacy code.
+  - `modules:<umd|amd|commonjs|systemjs|false>` allows you to control the output module format. Setting this value to `false` will disable the transformation of output module format altogether, keeping `import` and `export` statements largely intact.
+  - `es2017:<true|false>` toggles support for compiling code ES2017 code. When ES2017 is ratified this flag will be removed and the ES2017 preset will be added by default.
```

Comment:
Actually, thinking about this, I'd like to change this (and the `add-module-exports` flag) to not need a value *at all*. What I mean by that is instead of `--flags es2017:true` or `--flags add-module-exports:true` you'd just do `--flags es2017` or `--flags add-module-exports`. To clarify intent, maybe rename the `es2017` to `enable-es2017`. Other presets we may include could be represented the same way, like `enable-stage-0` etc. Thoughs?